### PR TITLE
[fix] Volume control moves accordingly to mousewheel event on Firefox

### PR DESCRIPTION
**Pull request by ysa000**
Properties in wheelevent named differently in Chrome & Firefox. Function modified accordingly so the volume control with the mousewheel works on both browsers.

### CODE REVIEW

File: app/views/playerControlsExtended.js [hunk 1]
```diff
@@ -104,9 +104,16 @@ const ControlsExtended = Mn.ItemView.extend({
     changeVolWheel(e) {
         const wheelStep = 0.05;
         let volume = application.appState.get('currentVolume', volume);
-        if (e.originalEvent.wheelDelta > 0 ) {
-            volume += wheelStep;
+        let wheeldelta;
+        if (/firefox/i.test(window.navigator.userAgent)) {
```
Comment (m4dz) [inline app/views/playerControlsExtended.js]: Sniffing userAgent isn't really reliable, as we may assume the user agent isn't lying on itself.

I suggest we find the delta by doing:

```js
const delta = e.originalEvent.wheelDelta || -e.originalEvent.deltaY
```

It's a smarter solution to support both mouseWheel interface and legacy DomScrollWheel 😉

Comment (fabienmorin) [reply app/views/playerControlsExtended.js]: May this [doc](https://www.h3xed.com/programming/javascript-mouse-scroll-wheel-events-in-firefox-and-chrome) can help ?

**Comment by m4dz** [comment]
👍

**Comment by ysa000** [comment]
@m4dz,
Thanks for the tip 👍 !

**Pull request merged**